test/images/agnhost/net/main.go

```diff
@@ -49,8 +50,33 @@ type logOutput struct {
 	b bytes.Buffer
 }
 
-func main() {
-	initFlags()
+// CmdNet is used by agnhost Cobra.
+var CmdNet = &cobra.Command{
+	Use:   "net",
+	Short: "TBA",
+	Long:  "TBA",
```

Comment:
Still needs written. Just copy from the readmes?

Reply:
Indeed. I didn't have the README written when I added these, I wanted to test first. :)